polaris-react/src/components/Banner/tests/Banner.test.tsx

```diff
@@ -22,410 +18,14 @@ import {UnstyledButton} from '../../UnstyledButton';
 import {UnstyledLink} from '../../UnstyledLink';
 import {BannerContext} from '../../../utilities/banner-context';
 import {WithinContentContext} from '../../../utilities/within-content-context';
-import {Banner} from '../Banner';
-import type {BannerHandles, BannerStatus} from '../Banner';
-// eslint-disable-next-line @shopify/strict-component-boundaries -- se23 test will eventaully be beside component
 import {
+  Banner,
   DefaultBanner,
   InlineIconBanner,
   WithinContentContainerBanner,
-} from '../components/BannerExperimental/BannerExperimental';
-
-describe('<Banner />', () => {
```

Comment:
These were the tests for the old `Banner`, the tests for the new one are in a different describe block below